bin/dev-tools/create_shipments.py

```diff
@@ -0,0 +1,332 @@
+#! /usr/bin/env python3
+
+import argparse
+import sys
+import json
+from json.decoder import JSONDecodeError
+from datetime import datetime, timezone, timedelta
+from uuid import uuid4
+
+from copy import deepcopy
+from random import randint
+import requests
+
+# pylint:disable=invalid-name
+parser = argparse.ArgumentParser()
+parser.add_argument("--total", "-t", help="Total amount of shipments to be created, defaults to 10", type=int)
+parser.add_argument("--startnumber", "-n", help="Number to start at for sequential attributes", type=int)
+parser.add_argument("--partition", "-p", help="Number shipments to create per wallet, defaults to 10", type=int)
+parser.add_argument("--carrier", "-c", help="Set carrier wallet owner (defaults to user 1)")
+parser.add_argument("--shipper", "-s", help="Set shipper wallet owner (defaults to user 1)")
+parser.add_argument("--moderator", "-m", help="Set moderator wallet owner (defaults to None)")
+parser.add_argument("--verbose", "-v", help="More descriptive when running functions.", action="store_true")
+parser.add_argument("--device", "-d", help="Add devices to shipment creation, defaults to false",
+                    action="store_true")
+parser.add_argument("--attributes", "-a", action='append',
+                    help="Attributes to be sent in shipment creation (defaults to required values)")
+parser.add_argument("--add_tracking", help="Adds tracking data to shipments (requires --device or -d)",
+                    action='store_true')
+parser.add_argument("--add_telemetry", help="Adds telemetry data to shipments (requires --device or -d)",
+                    action='store_true')
+
+# Keep all but the first command-line arguments
+argument_list = sys.argv[1:]
+
+
+# pylint:disable=too-many-instance-attributes
+class CreateShipments:
+    # Default variables for local use
+    profiles_url = 'http://localhost:9000'
+    transmission_url = 'http://localhost:8000'
```

Comment:
Yeah I was going to wait for that until it was more requested as it would require signing the tracking data and such and require more changes. I do feel like it would make more sense maybe to include all the urls and then instead of specifying a specific url, specify an environment. That would be better flow since the transmission/profiles would both need to be changed and would require just one variable instead of two.

Reply:
If I wanted to run this within the docker container, I currently cannot do that.  I would need to be able to specify the appropriate docker host for the services.

Reply:
So, would it make sense then to have the option to either set them manually or in a later version set them by calling the env name?